test/screenshotter/ss_data.json

```diff
@@ -4,6 +4,7 @@
     "Baseline": "http://localhost:7936/test/screenshotter/test.html?m=a+b-c\\cdot d/e",
     "BasicTest": "http://localhost:7936/test/screenshotter/test.html?m=a",
     "BinomTest": "http://localhost:7936/test/screenshotter/test.html?m=\\dbinom{a}{b}\\tbinom{a}{b}^{\\binom{a}{b}+17}",
+    "Cases": "http://localhost:7936/test/screenshotter/test.html?m=f%28a%2Cb%29%3D\\begin{cases}a%2B1%26\\text{if%20}b\\text{%20is%20odd}\\\\a%26\\text{if%20}b%3D0\\\\a-1%26\\text{otherwise}\\end{cases}",
```

Comment:
You don't need to escape all of these characters. For instance, you can put literal spaces instead of `%20`, literal parentheses, etc. I think that'll make this test make a bit more sense. You can even put literal `=` signs.